File: app/src/main/java/com/example/androidproject/FinalProjectApplication.java
package com.example.androidproject;

import android.app.Application;
import android.content.Context;

public class FinalProjectApplication extends Application {
    static Context context;
    public static Context getMyContext(){
        return context;
    }

    @Override
    public void onCreate() {
        super.onCreate();
        context = getApplicationContext();
    }
}

File: app/src/main/java/com/example/androidproject/FinalProjectApplication.kt
package com.example.androidproject

import android.app.Application
import com.example.androidproject.data.room.AppDatabase
import com.example.androidproject.utils.CloudinaryHelper

class FinalProjectApplication : Application() {
    override fun onCreate() {
        super.onCreate()

        // Initialize Cloudinary
        CloudinaryHelper.init(this)

        // Initialize Room database
        AppDatabase.getDatabase(this)
    }
}